lxd/main_forkusernsexec.go

```diff
@@ -233,14 +273,16 @@ void forkusernsexec(void)
 import "C"
 
 type cmdForkusernsexec struct {
-	global *cmdGlobal
+	global     *cmdGlobal
+	keepFdUpTo int
 }
 
 func (c *cmdForkusernsexec) Command() *cobra.Command {
 	// Main subcommand
 	cmd := &cobra.Command{}
 	cmd.Use = "forkusernsexec <cmd> <args>"
 	cmd.Short = "Run command in user namespace"
+	cmd.Flags().IntVar(&c.keepFdUpTo, "keep-fd-up-to", (1 << 31), fmt.Sprintf("Keep all fds below this one and close all the ones above it"))
```

Comment:
This comment doesn't define what happens to the number specified in the argument, only what happens to the fds below it and above it. I think tweak it to say something like:

"Keep all fds below and including this one and close all the ones above it"

Reply:
done